frontend/public/components/resource-quota.jsx

```diff
@@ -453,10 +586,43 @@ export const ResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(
   },
 );
 
-export const ResourceQuotasDetailsPage = (props) => (
-  <DetailsPage
-    {...props}
-    menuActions={resourceQuotaMenuActions}
-    pages={[navFactory.details(Details), navFactory.editYaml()]}
-  />
+export const AppliedClusterResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(
```

Comment:
We're connecting to flags here, but I don't see how the flag is used. If you're already on the AppliedClusterResourceQuota page, it should only be OpenShift.